src/components/CameraPermissions/Primer/style.css

```diff
@@ -35,15 +57,10 @@
   background-image: url('assets/allow.svg');
   background-repeat: no-repeat;
   background-size: 100%;
-  height: 121*@unit-small;
-  margin: 0 auto 30*@unit-small;
   position: relative;
-  width: 270*@unit-small;
-
-  @media (--small-viewport) {
-    height: 100*@unit-small;
-    width: 224*@unit-small;
-  }
+  flex: 0 1 100%;
+  /* aspect ratio of responsive div */
+  padding-top: 44%;
```

Comment:
where does the 44% come from? some calculation or eyeballing?

Reply:
The original height value divided by the original width value. 121/270 = 0.44

Reply:
can you put that in the comment? it gives some clues of where it comes from

Reply:
Good point. Will add 👍